it/controllers/VatRegistrationControllerISpec.scala

```diff
@@ -89,4 +89,18 @@ class VatRegistrationControllerISpec extends IntegrationStubbing with FeatureSwi
     }
   }
 
+  "PATCH  /:regId/honesty-declaration" should {
+    "return Ok if the honesty declaration is successfully stored" in new Setup {
+      given.user.isAuthorised
+
+      insertIntoDb(testEmptyVatScheme("regId"))
+
+      val res = await(client(controllers.routes.VatRegistrationController.storeHonestyDeclaration(testRegId).url)
+      .patch(Json.obj("honestyDeclaration" -> "testValue"))
+      )
+
+      res.status mustBe OK
+    }
```

Comment:
You could test that the db has been updated by pulling the new data out and do a comparison